### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by himadri2110**
Fixes:

- Updated the meta tags of the app
- Added a `_redirects` file to fix the page refresh issue
- Added categories on the home page
- Clicking on the home page category redirects to the filters page with that filter applied
- Login with test credentials
- Error handling in login and sign up 
- Out of Stock card styling
- Fixed click on wishlist/cart icon when logged out

### CODE REVIEW

**Review by bipin7yadav — APPROVED** [review]
good to go

File: src/components/ProductListing/ProductListing.css [hunk 1]
```diff
@@ -20,11 +20,11 @@
...
-  .product-wrapper {
-    padding: 1rem;
-  }
+.product-heading .fa-sliders {
+  transform: rotate(90deg);
+}
 
+@media screen and (max-width: 650px) {
   .product-main {
     grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
```
Comment (Chakravarthi0) [inline src/components/ProductListing/ProductListing.css:29]: I would suggest you to use rem here.

File: src/pages/Home/Home.css [hunk 2]
```diff
@@ -107,10 +138,23 @@
   .hero-content .sub-title {
     font-size: 1rem;
   }
+
+  .category-grid {
+    grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
```
Comment (Chakravarthi0) [inline src/pages/Home/Home.css:143]: You can use rem here as well

File: src/pages/Home/Home.css [hunk 3]
```diff
@@ -107,10 +138,23 @@
...
+    object-fit: contain;
+  }
 }
 
 @media screen and (max-width: 480px) {
   .card-wrapper .row {
     flex-direction: column;
   }
+  .category-grid {
+    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```
Comment (Chakravarthi0) [inline src/pages/Home/Home.css:158]: and here too

**Review by Chakravarthi0 — APPROVED** [review]
Clean code,
one suggestion: use rem unit instead of px unit. 


Rest looks good.

**Pull request merged**